pages/common/xidel.md

```diff
@@ -0,0 +1,36 @@
+# xidel
+
+> A command line tool to download and extract data from HTML/XML pages as well as JSON APIs.
+> More information: <https://www.videlibri.de/xidel.html>.
+
+- Print all URLs found by a google search:
```

Comment:
I feel like a number of the examples on this page are too specific to a single website. Can we generalise them at all?

Reply:
I'd like to add, the google links are .de, rather than a default google.com

Reply:
Not sure how to work with first comment.  I took the examples straight from the main website.  3-5 seem to be the best basic ones to use.